test/forcing/NetCDFPerFeatureDataProvider_Test.cpp

```diff
@@ -61,41 +72,46 @@ TEST_F(NetCDFPerFeatureDataProviderTest, TestForcingDataRead)
 {
     // check to see that the variable "T2D" exists
     auto var_names = nc_provider->get_avaliable_variable_names();
-    auto pos = std::find(var_names.begin(), var_names.end(), "T2D");
+    auto pos = std::find(var_names.begin(), var_names.end(), CSDMS_STD_NAME_SURFACE_TEMP);
 
     if ( pos != var_names.end() )
     {
-        std::cout << "Found variable T2D\n";
+        std::cout << "Found variable "<<CSDMS_STD_NAME_SURFACE_TEMP<<"\n";
     }
     else
     {
-        std::cout << "The variable \"T2D\" is missing\n";
+        std::cout << "The variable "<<CSDMS_STD_NAME_SURFACE_TEMP<<" is missing\n";
         FAIL();
     }
 
     auto start_time = nc_provider->get_data_start_time();
     auto ids = nc_provider->get_ids();
     auto duration = nc_provider->record_duration();
 
+    std::cout << "Checking values in catchment "<<ids[0]<<" at time "<<start_time<<" ..."<<std::endl;
+
     // read exactly one time step correctly aligned
-    double val1 = nc_provider->get_value(NetCDFDataSelector(ids[0], "T2D", start_time, duration, "K"), data_access::MEAN);
+    double val1 = nc_provider->get_value(NetCDFDataSelector(ids[0], CSDMS_STD_NAME_SURFACE_TEMP, start_time, duration, "K"), data_access::MEAN);
+
+    //double tol = 0.00000612;
+    double tol = 0.001;
```

Comment:
That might be to high of an acceptable floating point error value

Reply:
Got this down to 0.00002 in latest commit. That seems more acceptable for NetCDF `float`.